packages/client/test/integration/test-node/test/main.spec.ts

```diff
@@ -1,87 +1,116 @@
-import { Torii } from '@iroha2/client'
-import { type RustResult, datamodel, sugar, variant } from '@iroha2/data-model'
+import { type datamodel, sugar} from '@iroha2/data-model'
 import { CLIENT_CONFIG } from '@iroha2/test-configuration'
 import { Seq } from 'immutable'
-import { describe, expect, test } from 'vitest'
+import { expect, test, beforeEach } from 'vitest'
 import { clientFactory, setupPeerTestsLifecycle } from './util'
 import { pipe } from 'fp-ts/function'
+import * as allure from 'allure-vitest'
 
 setupPeerTestsLifecycle()
 
-// Actually, it is already tested within `@iroha2/test-peer`
-test('Peer is healthy', async () => {
-  const { pre } = clientFactory()
-
-  expect(await Torii.getHealth(pre)).toEqual(variant('Ok', null) as RustResult<null, any>)
+beforeEach(async (ctx) =>{
+  await allure.owner(ctx, "dulger");
+  await allure.label(ctx, "permission", "no_permission_required");
+  await allure.label(ctx, "sdk", "Java Script");
 })
 
-test('AddAsset instruction with name length more than limit is not committed', async () => {
+test('Register asset with too long type', async (ctx) => {
+  await allure.allureId(ctx, "3739");
+  await allure.label(ctx, "feature", "Assets");
+  await allure.label(ctx, "sdk_test_id", "register_asset_with_too_long_type");
+  await allure.label(ctx, "story", "Account registers an asset with too long type");
+
   const { client, pre, getBlocksListener } = clientFactory()
   const blocks = await getBlocksListener()
 
   const normalAssetDefinitionId = sugar.assetDefinitionId('xor', 'wonderland')
 
-  const tooLongAssetName = '0'.repeat(2 ** 14)
-  const invalidAssetDefinitionId = sugar.assetDefinitionId(tooLongAssetName, 'wonderland')
-
-  await blocks.wait(async () => {
-    await Promise.all(
-      // we should register these assets as separate transactions, because the invalid
-      // one will be rejected
-      [normalAssetDefinitionId, invalidAssetDefinitionId].map(async (id) => {
-        await client.submitExecutable(
-          pre,
+  let tooLongAssetName;
+  let invalidAssetDefinitionId;
 
-          pipe(
-            sugar.identifiable.newAssetDefinition(id, datamodel.AssetValueType('BigQuantity')),
-            sugar.instruction.register,
-            sugar.executable.instructions,
-          ),
-        )
-      }),
-    )
+  await allure.step(ctx, "Creating asset with long asset name", async()=>{
```

Comment:
This step is meaningless. It is a plain JS operation and is not related to allure & compat matrix.